card/mod/machines/lib/javascript/decko/components.js.coffee

```diff
@@ -13,10 +15,7 @@ $(window).ready ->
       type: 'GET'
       success: (html) ->
         $(tab_id).append(html)
-        $(tab_id).triggerSlotReady()
-
-  $('body').on "select2:select", "._submit-on-select", (event) ->
-    $(event.target).closest('form').submit()
+        decko.contentLoaded($(tab_id), $(this))
```

Comment:
`contentLoaded` is the new starting point for all the slot initialization stuff. I didn't name it "slotLoaded" because it is also called for modals and overlays.